commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java

```diff
@@ -0,0 +1,23 @@
+package io.sphere.sdk.products;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import static org.apache.commons.lang3.StringUtils.capitalize;
+
+/**
+ * PublishScope.
+ */
+public enum PublishScope {
```

Comment:
Sorry that I forgot it in my last comments: Each enum in our SDK must implement the `SphereEnumeration` interface.